spec/controllers/dashboard_controller_spec.rb

```diff
@@ -17,11 +17,11 @@
       render_views
 
       before do
+        stub_auth_token
         stub_poms(prison, poms)
-        stub_sso_pom_data(prison)
+        stub_sso_pom_data(prison, 'alice')
         stub_signed_in_pom(1, 'Alice')
-        stub_request(:get, "https://gateway.t3.nomis-api.hmpps.dsd.io/elite2api/api/users/").
-          with(headers: { 'Authorization' => 'Bearer token' }).
+        stub_request(:get, "https://gateway.t3.nomis-api.hmpps.dsd.io/elite2api/api/users/alice").
```

Comment:
I think you should now be able to delete this line - it should already be done by the stub_sso_pom_data